src/components/frame/HamburgerDropdown/HamburgerDropdownPopover.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+import DropdownMenu, {
+  DropdownMenuItem,
+  DropdownMenuSection,
+} from '~shared/DropdownMenu';
+import {
+  ColonySection,
+  HelperSection,
+  MetaSection,
+  UserSection,
+} from '~shared/PopoverSection';
+import NavLink from '~shared/NavLink';
+import { useAppContext, useColonyContext } from '~hooks';
+
+import styles from './HamburgerDropdownPopover.css';
+
+const displayName = 'frame.HamburgerDropdown.HamburgerDropdownPopover';
+
+const MSG = defineMessages({
+  actions: {
+    id: `${displayName}.actions`,
+    defaultMessage: 'Actions',
+  },
+  funds: {
+    id: `${displayName}.funds`,
+    defaultMessage: 'Funds',
+  },
+  members: {
+    id: `${displayName}.members`,
+    defaultMessage: 'Members',
+  },
+  extensions: {
+    id: `${displayName}.extensions`,
+    defaultMessage: 'Extensions',
+  },
+  buyTokens: {
+    id: `${displayName}.buyTokens`,
+    defaultMessage: 'Buy Tokens',
+  },
+});
+
+interface Props {
+  closePopover: () => void;
+}
+
+const HamburgerDropdownPopover = ({ closePopover }: Props) => {
+  const { colony } = useColonyContext();
+  const { user, wallet } = useAppContext();
+  const colonyHomePath = `/colony/${colony?.name}`;
+
+  return (
+    <div className={styles.menu}>
+      <DropdownMenu onClick={closePopover}>
+        {user?.name && colony?.name && (
+          <DropdownMenuSection>
+            <DropdownMenuItem>
+              <NavLink to={colonyHomePath} text={MSG.actions} />
+            </DropdownMenuItem>
+            <DropdownMenuItem>
+              <NavLink to={`${colonyHomePath}/funds`} text={MSG.funds} />
+            </DropdownMenuItem>
+            <DropdownMenuItem>
+              <NavLink to={`${colonyHomePath}/members`} text={MSG.members} />
+            </DropdownMenuItem>
+            <DropdownMenuItem>
+              <NavLink
+                to={`${colonyHomePath}/extensions`}
+                text={MSG.extensions}
+              />
+            </DropdownMenuItem>
+          </DropdownMenuSection>
```

Comment:
Consider wrapping this like the others? I guess really this should be `ColonySection` and the other one `CreateColonySection`, or similar.

Reply:
Thanks Will! I made the changes you suggested. 👍